lib/active_admin/resource/attributes.rb

```diff
@@ -0,0 +1,30 @@
+module ActiveAdmin
+
+  class Resource
+    module Attributes
+
+      def default_attributes
+        resource_class.columns.each_with_object({}) do |c, attrs|
+          #skip primary key, STI column and counter cache columns
+          unless  c.name == resource_class.primary_key || c.name == resource_class.inheritance_column || c.name =~ /_count$/
```

Comment:
I would extract `primary_key_col?` `sti_col?` and `counter_cache_col?` methods here. That way we avoid the need for the comment since the code would be self-explanatory.